packages/checkbox-group/src/vaadin-checkbox-group.d.ts

```diff
@@ -116,7 +116,7 @@ declare class CheckboxGroupElement extends ThemableMixin(DirMixin(HTMLElement))
 
   _removeCheckboxFromValue(value: string): void;
 
-  _changeSelectedCheckbox(checkbox: CheckboxElement | null): void;
+  _changeSelectedCheckbox(checkbox: Checkbox | null): void;
```

Comment:
Let's consider removing these methods in the PR where you will update `vaadin-checkbox-group` itself.